depmapomics/ccle.py

```diff
@@ -0,0 +1,1423 @@
+        mode="bool",
+        mutNameCol="mutname",
+        minfreqtocall=minfreqtocall,
+    ).astype(int).T.to_csv(
+        folder + "somatic_mutations_boolmatrix_fordepmap_damaging.csv"
+    )
+    mut.mafToMat(
+        merged[(merged.Variant_annotation == "other conserving")],
+        mode="bool",
+        mutNameCol="mutname",
+        minfreqtocall=minfreqtocall,
+    ).astype(int).T.to_csv(
+        folder + "somatic_mutations_boolmatrix_fordepmap_othercons.csv"
+    )
+    mut.mafToMat(
+        merged[(merged.Variant_annotation == "other non-conserving")],
+        mode="bool",
+        mutNameCol="mutname",
+        minfreqtocall=minfreqtocall,
+    ).astype(int).T.to_csv(
+        folder + "somatic_mutations_boolmatrix_fordepmap_othernoncons.csv"
+    )
+    mut.mafToMat(
+        merged[(merged.isCOSMIChotspot | merged.isTCGAhotspot)],
+        mode="bool",
+        mutNameCol="mutname",
+        minfreqtocall=minfreqtocall,
+    ).astype(int).T.to_csv(
+        folder + "somatic_mutations_boolmatrix_fordepmap_hotspot.csv"
+    )
+
+    # uploading to taiga
+    tc.update_dataset(
+        changes_description="new " + samplesetname + " release!",
+        dataset_permaname=taiga_dataset,
+        upload_files=[
+            # for depmap
+            {
+                "path": folder + "somatic_mutations_boolmatrix_fordepmap_hotspot.csv",
+                "name": "somatic_mutations_boolmatrix-hotspot-profile",
+                "format": "NumericMatrixCSV",
+                "encoding": "utf-8",
+            },
+            {
+                "path": folder
+                + "somatic_mutations_boolmatrix_fordepmap_othernoncons.csv",
+                "name": "somatic_mutations_boolmatrix-othernoncons-profile",
+                "format": "NumericMatrixCSV",
+                "encoding": "utf-8",
+            },
+            {
+                "path": folder + "somatic_mutations_boolmatrix_fordepmap_damaging.csv",
+                "name": "somatic_mutations_boolmatrix-damaging-profile",
+                "format": "NumericMatrixCSV",
+                "encoding": "utf-8",
+            },
+            # {
+            #     "path": folder + "somatic_mutations_boolmatrix_fordepmap_othercons.csv",
+            #     "name": "somatic_mutations_boolmatrix-othercons-profile",
+            #     "format": "NumericMatrixCSV",
+            #     "encoding": "utf-8",
+            # },
+            {
+                "path": folder + "somatic_mutations_fordepmap_profile.csv",
+                "name": "somatic_mutations-profile",
+                "format": "TableCSV",
+                "encoding": "utf-8",
+            },
+            {
+                "path": "output/" + samplesetname + "/wes_somatic_mutations_all.csv",
+                "format": "TableCSV",
+                "encoding": "utf-8",
+            },
+            {
+                "path": "output/" + samplesetname + "/wgs_somatic_mutations_all.csv",
```

Comment:
same no distinguishing between wes/wgs. Also let's not do a specific fordepmap vs the rest (the fordepmap filters are applied to the profile level one but not the cds level one